### DISCUSSION TITLE
docker instructions missing mount

### DISCUSSION THREAD

**Discussion by llllllllillllllillll** [tabler/tabler#1450] [Ideas]
`_config_prod.yml` mount is missing from the docker instructions.

instructions should be : 

`docker run -p 3000:3000 -p 3001:3001 -v $(pwd)/src:/app/src -v $(pwd)/_config.yml:/app/_config.yml -v $(pwd)/_config_prod.yml:/app/_config_prod.yml tabler`

**Comment by s0apye** [comment]
Thanks dude, that worked for me.
